src/test/java/core/basesyntax/LambdaCalculatorTest.java

```diff
@@ -0,0 +1,341 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class LambdaCalculatorTest {
+    private static Calculator calculator;
+    private static final double DELTA = 0.0001;
```

Comment:
```suggestion
    private static final double DELTA = 0.0001;
    private static Calculator calculator;
    
```

we should pleace constants first